src/js/controller/controller.js

```diff
@@ -554,6 +554,7 @@ Object.assign(Controller.prototype, {
                 if (instream) {
                     instream.noResume = true;
                 }
+                _actionOnAttach = () => _programController.stopVideo();
```

Comment:
`_actionOnAttach` is set to null when pause is called https://github.com/jwplayer/jwplayer/blob/993215ceef26f1fbf5c59317948a21a63115d502/src/js/controller/controller.js#L578

Reply:
That's good. I hope it's also set to null on any call that would take the player out of the idle state (starting after stopping)

Reply:
what i meant is that if you call stop during an ad, and then call pause, the video won't stop when the ad ends. is that expected behavior ?

Reply:
`pause()` shouldn't do anything after calling `stop()`, so what is it doing @karimMourra ?

Reply:
@robwalch If a developer calls stop during an ad, he expects the player to go to idle state when the ad ends. By calling pause afterwards, a viewer can prevent the stop from happening. If a developer wants to ensure that the video is stopped after the ad, he has to add an onAdPause listener and call stop again ?

Reply:
@karimMourra: @alkupe added that line in 2013. Maybe you and @vseventer could investigate removing it?

Reply:
<img width="568" alt="screen shot 2019-02-26 at 7 48 17 pm" src="https://user-images.githubusercontent.com/333258/53457208-81490200-39ff-11e9-916c-efac0d095ebb.png">